packages/terra-markdown/src/Markdown.jsx

```diff
@@ -19,7 +19,7 @@ const defaultProps = {
 const Markdown = props => (
   <div
     dir="ltr"
-    className="markdown-body"
+    className={'markdown-body'}
```

Comment:
Markdown will probably need to be done separately because it could be a fair amount of work. Markdown is currently importing [github-markdown-css](https://github.com/cerner/terra-core/blob/master/packages/terra-markdown/src/Markdown.jsx#L5)... this will probably need to be rewritten.